packages/lib/src/core/core.component.props.test.ts

```diff
@@ -4,6 +4,11 @@ import { PaymentMethodsResponse } from '../types';
 import { PayPal, Card, Dropin, Redirect } from '../components';
 import { CoreConfiguration } from './types';
 
+import getTranslations from './Services/get-translations';
+jest.mock('./Services/get-translations');
+const mockedGetTranslations = getTranslations as jest.Mock;
+mockedGetTranslations.mockResolvedValue({});
+
```

Comment:
Not sure since this is not simply an object that is used around, but it is actually mocking an import . 
Perhaps it should be possible to mock it for all tests, although sometimes it is nice to make things more explicit.. I will take a look if it can be improved